teeio-validator/include/ide_test.h

```diff
@@ -516,11 +540,22 @@ struct _ide_run_test_case {
   ide_common_test_case_teardown_func_t teardown_func;
 };
 
+typedef struct {
+  uint32_t signature;
+  void *group_context;
+
+  ide_run_test_case_t* test_case;
+
+  IDE_COMMON_TEST_ACTION action;
+  IDE_COMMON_TEST_CASE_RESULT result;
+} ide_common_test_case_context_t;
+
 typedef struct {
   ide_common_test_case_setup_func_t setup;
   ide_common_test_case_run_func_t run;
   ide_common_test_case_teardown_func_t teardown;
   bool config_check_required;
+  int assertion_cnt;
```

Comment:
why need assertion_cnt?

Why not calculate it at runtime, as SPDM-Responder-Validator?

Reply:
Yes, assertion_cnt is not needed. It will be deleted in next version.